src/ufoLib2/objects/misc.py

```diff
@@ -148,7 +183,8 @@ def write(self, writer, saveAs=None):
             # 1) We are saving in-place. Only write to disk what is loaded, it
             #    might be modified.
             # 2) We save elsewhere. Load all data files to write them back out.
-            if data is _NOT_LOADED:
+            # XXX: Move write_data into `if saveAs` branch to simplify code?
+            if isinstance(data, Placeholder):
```

Comment:
Why did you introduce this Placeholder class and change the `is _NOT_LOADED` to `isinstance`? Is it all to appease typing problems?

Reply:
yes. mypy recognizes `isinstance`, but not `is`

Reply:
(One could use `typing.cast` though)

Reply:
Ok, I think the isInstance check works the same, but it's a shame that mypy doesn't recognize the other pattern, it's a very common and useful way of implementing sentinel values

Reply:
yes shame